src/app/wallet/wallet/send/send.component.ts

```diff
@@ -68,13 +72,26 @@ export class SendComponent implements OnInit {
     this._rpcState.observe('getblockchaininfo', 'chain').pipe(take(1))
       .subscribe(chain => this.testnet = chain === 'test');
 
-    this.sendService.listUnSpent();
+    // Calculate Spendable balance
+    this._rpcState.observe('listunspent')
+      .pipe(takeWhile(() => !this.destroyed))
+      .subscribe(
+        unspent => {
+          this.availableBal = 0;
```

Comment:
Create a temporary variable of type number here (default assigned a value of 0), then set this.availableBal after the for-loop below, using the temp variable to create an Amount type.